src/commands/importWebApp/importWebApp.ts

```diff
@@ -0,0 +1,321 @@
+}
+
+export async function listWebApps(client: WebSiteManagementClient, webAppType: string): Promise<Site[]> {
+    const allWebApps: WebAppCollection = await client.webApps.list();
+    if ((webAppType === "webApp")) {
+        return allWebApps.filter((ele) => !!ele.kind && !ele.kind.includes("functionapp"));
+    }
+    return allWebApps.filter((ele) => !!ele.kind && ele.kind.includes("functionapp"));
+}
+
+export async function pickWebApp(apiApps: Site[]): Promise<Site> {
+    // Pick function app
+    const apiApp = await ext.ui.showQuickPick(apiApps.map((s) => { return { label: nonNullOrEmptyValue(s.name), site: s }; }), { canPickMany: false });
+    return apiApp.site;
+}
+
+// Create policy for importing
+export function createImportXmlPolicy(backendId: string): string {
+    const operationPolicy = [{
+        policies: [
+            {
+                inbound: [
+                    { base: null },
+                    {
+                        "set-backend-service": [
+                            {
+                                _attr: {
+                                    id: "apim-generated-policy",
+                                    "backend-id": backendId
+                                }
+                            }
+                        ]
+                    }
+                ]
+            },
+            { backend: [{ base: null }] },
+            { outbound: [{ base: null }] },
+            { "on-error": [{ base: null }] }
+        ]
+    }];
+
+    return xml(operationPolicy);
+}
+
+// Create backend for the web app
+export async function setAppBackendEntity(node: ApisTreeItem | ApiTreeItem, backendId: string, appName: string, appPath: string, appResourceGroup: string, webAppName: string, BackendCredentials?: BackendCredentialsContract): Promise<void> {
+    const nbackend: BackendContract = {
+        description: `${appName}`,
+        resourceId: `${node.root.environment.resourceManagerEndpointUrl}/subscriptions/${node.root.subscriptionId}/resourceGroups/${appResourceGroup}/providers/Microsoft.Web/sites/${webAppName}`,
+        url: appPath,
+        id: backendId,
+        name: backendId,
+        protocol: "http",
+        credentials: BackendCredentials
+    };
+    await node.root.client.backend.createOrUpdate(node.root.resourceGroupName, node.root.serviceName, backendId, nbackend);
+}
+
+// Create new api from web app
+export async function constructApiFromWebApp(apiId: string, webApp: Site, apiName: string): Promise<ApiContract> {
+    return {
+        description: `Import from "${webApp.name}" Function App`,
+        id: apiId,
+        name: apiName,
+        displayName: apiName,
+        path: "",
+        protocols: ["https"]
+    };
+}
+
+async function importFromSwagger(webAppConfig: IWebAppContract, webAppName: string, apiName: string, node: ApiTreeItem | ApisTreeItem): Promise<void> {
+    if (webAppConfig.properties.apiDefinition && webAppConfig.properties.apiDefinition.url) {
```

Comment:
This check is already done in the caller (webAppConfig.properties.apiDefinition && webAppConfig.properties.apiDefinition.url)